src/content/en/updates/posts/2016/03/access-usb-devices-on-the-web.markdown

```diff
@@ -0,0 +1,295 @@
+---
+layout: updates/post
+title: "Access USB devices on the Web"
+description: "The WebUSB API has been created to provide an easy way to safely expose USB device services to the Web."
+featured_image: /web/updates/images/2016-03-02-access-usb-devices-on-the-web/web-usb-hero.jpg
+published_on: 2016-03-02
+updated_on: 2016-03-30
+authors:
+  - beaufortfrancois
+tags:
+  - news
+  - USB
+  - WebUSB
+  - IoT
+  - Arduino
+---
+
+If I tell you plain and simple "USB", there are good chances that you will
+immediately think of keyboards, mice, audio, video and storage devices. You're
+right but you'll find other kinds of Universal Serial Bus (USB) devices out
+there.
+
+These non-standardized USB devices require hardware vendors to write native
+drivers and SDKs in order for you (developer) to take advantage of them. Sadly
+this native code has historically prevented these devices from being used by
+the Web. And that's exactly why the WebUSB API has been created: to **provide
+an easy way to safely expose USB device services to the Web**. With this API,
+hardware manufacturers will be able to build cross-platform JavaScript SDKs for
```

Comment:
It is because the hardware manufacturer **has to** enable WebUSB indeed. Not any USB device can be controlled from the Web. They have to manufactured for that.
This is a major difference with Web Bluetooth.